pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -0,0 +1,391 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2020  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "PrecompiledHeader.h"
+
+#include "ATA.h"
+#include "../DEV9.h"
+
+void WriteUInt16(u8* data, int* index, u16 value)
+{
+	*(u16*)&data[*index] = value;
+	*index += sizeof(value);
+}
+void WriteUInt32(u8* data, int* index, u32 value)
+{
+	*(u32*)&data[*index] = value;
+	*index += sizeof(value);
+}
+void WriteUInt64(u8* data, int* index, u64 value)
+{
+	*(u64*)&data[*index] = value;
+	*index += sizeof(value);
+}
+//No null char
+void WritePaddedString(u8* data, int* index, std::string value, u32 len)
+{
+	memset(&data[*index], (u8)' ', len);
+	memcpy(&data[*index], value.c_str(), value.length() < len ? value.length() : len);
+	*index += len;
+}
+
+void ATA::CreateHDDinfo(int sizeMb)
+{
+	const u16 sectorSize = 512;
+	DEV9_LOG_VERB("HddSize : %i\n", config.HddSize);
+	const u64 nbSectors = ((u64)(sizeMb / sectorSize) * 1024 * 1024);
+	DEV9_LOG_VERB("nbSectors : %i\n", nbSectors);
```

Comment:
Please use Console.Devlog for things like this

Reply:
Do you mean DevCon.WriteLn?

Reply:
That's what I meant indeed, brainfart

Reply:
Is this just for those lines? or elsewhere?

Reply:
There are quite a few occurences in DEV9, I suggest you to do a `grep -nr DEV9_LOG` to find all of them or equivalent

Reply:
Done, I've also replaced any emu_printf/printf/fprintf into Console.WriteLn/Error as I thought necceserry